src/app/styles.js

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+import colors from '../constants/colors';
+
+export default StyleSheet.create({
+  imageHeader: {
+    resizeMode: 'stretch',
+    width: '100%'
+  },
+  headerStyle: {
+    height: 102
+  },
+  headerCard: {
+    backgroundColor: colors.polar
+  },
+  headerTitle: {
+    color: colors.white,
+    fontSize: 17,
+    fontWeight: '600',
+    lineHeight: 20
+  },
+  backImage: {
+    maxHeight: 20,
+    maxWidth: 20
```

Comment:
if this constant value is needed to make it square, extract this value into a constant

Reply:
Done! I pushed them in a "globalStyle"